### ISSUE TITLE
"poetry install" fails on windows with demo project and base pyproject.toml

### ISSUE THREAD

**Issue by evenprimes** [python-poetry/poetry#2468]
---
name: "\U0001F41E Bug Report"
about: Did you find a bug?
title: '"poetry install" fails on windows with demo project and base pyproject.toml'
labels: Bug, Triage
assignees: ''

---

- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: Windows 10 v1909 build 18363.836
- **Poetry version**: 1.0.5 (installed via PowerShell script)
- **Python version**: 3.7.7 (installed via Windows store)
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: n/a - problem occurs with default pyproject.toml

## Issue

After installing and creating a bare demo project, `poetry install` fails. It looks like
it can't find the `sys` module.

### What I tried
* Uninstalled Python and installed fresh copy with poetry first/only Python module installed
* Tried `pip install poetry`, same error
* Also tried to install Python 3.8 (also from Windows store), same issue

### Other notes
I looked at [Virtualenv not always found in case insensitive filesystem, issue #2419](https://github.com/python-poetry/poetry/issues/2419), but the paths involved (see below) don't have spaces.

Both the Python 3.7 scripts folder and $home\.poetry\bin are in the system path.

This does not seem to be related to the [windows rmtree failures](https://github.com/python-poetry/poetry/pull/1032) previously reported.

Stack trace of failure:
```
PS C:\Users\didge\Source\poetry-demo> poetry install -vvv
[EnvCommandError]
Command ['C:\\Users\\didge\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\poetry-demo-2Mh1ooRm-py3.7\\Scripts\\python.exe', '-'] errored with the following return code 1, and output:
The system cannot find the path specified.
input was : import sys

if hasattr(sys, "real_prefix"):
    print(sys.real_prefix)
elif hasattr(sys, "base_prefix"):
    print(sys.base_prefix)
else:
    print(sys.prefix)


Traceback (most recent call last):
  File "C:\Users\didge\.poetry\lib\poetry\_vendor\py3.7\clikit\console_application.py", line 131, in run
    status_code = command.handle(parsed_args, io)
  File "C:\Users\didge\.poetry\lib\poetry\_vendor\py3.7\clikit\api\command\command.py", line 120, in handle
    status_code = self._do_handle(args, io)
  File "C:\Users\didge\.poetry\lib\poetry\_vendor\py3.7\clikit\api\command\command.py", line 163, in _do_handle
    self._dispatcher.dispatch(PRE_HANDLE, event)
  File "C:\Users\didge\.poetry\lib\poetry\_vendor\py3.7\clikit\api\event\event_dispatcher.py", line 22, in dispatch
    self._do_dispatch(listeners, event_name, event)
  File "C:\Users\didge\.poetry\lib\poetry\_vendor\py3.7\clikit\api\event\event_dispatcher.py", line 89, in _do_dispatch
    listener(event, event_name, self)
  File "C:\Users\didge\.poetry\lib\poetry\console\config\application_config.py", line 89, in set_env
    env = env_manager.create_venv(io)
  File "C:\Users\didge\.poetry\lib\poetry\utils\env.py", line 489, in create_venv
    env = self.get(reload=True)
  File "C:\Users\didge\.poetry\lib\poetry\utils\env.py", line 352, in get
    return VirtualEnv(venv)
  File "C:\Users\didge\.poetry\lib\poetry\utils\env.py", line 1022, in __init__
    self._base = Path(self.run('python', '-', input_=GET_BASE_PREFIX).strip())
  File "C:\Users\didge\.poetry\lib\poetry\utils\env.py", line 856, in run
    return self._run(cmd, **kwargs)
  File "C:\Users\didge\.poetry\lib\poetry\utils\env.py", line 1089, in _run
    return super(VirtualEnv, self)._run(cmd, **kwargs)
  File "C:\Users\didge\.poetry\lib\poetry\utils\env.py", line 893, in _run
    raise EnvCommandError(e, input=input_)
```

**Comment by rafaellehmkuhl** [comment]
Any news on this one? Same problem here.

**Comment by evenprimes** [comment]
I’ve just given up on using poetry for now, at least for projects hosted on windows. 

I’m considering trying it on the next project I host on a Pi.

**Comment by abn** [comment]
@rafaellehmkuhl @evenprimes does this also happen with the latest pre-release?

**Comment by rafaellehmkuhl** [comment]
> @rafaellehmkuhl @evenprimes does this also happen with the latest pre-release?

Didn't try. I'm using the latest stable. There was some change regarding this?

**Comment by cruzelante098** [comment]
I solved this problem by deleting the entire pypoetry folder.

```
rm -r "C:\Users\<Username>\AppData\Local\pypoetry"
```

**Comment by abn** [comment]
This issue could be related to #2651. The original error indicates that instead of the script being passed in as input argument, it was interpreted as a separate command.

This should be resolved in `1.1.0b1`.

**Comment by Satwikram** [comment]
I changed the registry values Autorun in HKEY_CURRENT_USER\Software\Microsoft\Command Processor from 
`if exists` to  `blank`. It worked in my case.
